conans/test/integration/toolchains/ios/_utils.py

```diff
@@ -0,0 +1,72 @@
+import textwrap
+
+lib_h = textwrap.dedent("""
+    #pragma once
+    #include <string>
+    class HelloLib {
+    public:
+        void hello(const std::string& name);
+    };
+""")
+
+lib_cpp = textwrap.dedent("""
+    #include "hello.h"
+    #include <iostream>
+    using namespace std;
+    void HelloLib::hello(const std::string& name) {
+        #ifdef DEBUG
+        std::cout << "Hello " << name << " Debug!" <<std::endl;
+        #else
+        std::cout << "Hello " << name << " Release!" <<std::endl;
+        #endif
+    }
+""")
+
+cpp_wrapper_h = textwrap.dedent("""
```

Comment:
Isn't it possible to have  a pure C++ library compiled for iOS?

That would be much better, if the testing code would be generic?. If we could be building exactly the same "hello world" C++ library with all toolchains, that would be great.

Reply:
It is not strictly needed, I just liked the idea of having also the wrapper there, that you will need if you want to call the library from Swift. If if makes it cleaner I'll remove it 👍